src/base/bittorrent/torrentinfo.cpp

```diff
@@ -292,7 +293,9 @@ QVector<QUrl> TorrentInfo::urlSeeds() const
 
     for (const lt::web_seed_entry &webSeed : nativeWebSeeds)
     {
+#if LIBTORRENT_VERSION_NUM < 20100
```

Comment:
I don't really like the conditional splitting the if block so I would duplicate the code.

Reply:
> I don't really like the conditional splitting the if block so I would duplicate the code.

OK. Fixed.